examples/language/llama/benchmark.py

```diff
@@ -307,7 +338,7 @@ def empty_init():
     torch.set_default_dtype(torch.bfloat16)
     model, optimizer, _, dataloader, _ = booster.boost(model, optimizer, dataloader=dataloader)
 
-    torch.set_default_dtype(torch.float)
+    # torch.set_default_dtype(torch.float)
```

Comment:
why?

Reply:
> why?

Add the set_default_dtype back in https://github.com/hpcaitech/ColossalAI/pull/6083/commits/982e4ee1f85a3aec285197d8bbdc3ca292bd15ff